### PR TITLE
[Editor] Display and Edit Resource Creation date

### PR DESCRIPTION
**Pull request by ronitjadhav**
### Description

This pull request adds the ability to display and edit the "Resource Created" date of the resource in the "About the resource" section.

### Screenshots

image

### Quality Assurance Checklist

- [ ] Commit history is devoid of any _merge commits_ and readable to facilitate reviews
- [ ] If **new logic** ⚙️ is introduced: unit tests were added
- [ ] If **new user stories** 🤏 are introduced: E2E tests were added
- [ ] If **new UI components** 🕹️ are introduced: corresponding stories in Storybook were created
- [ ] If **breaking changes** 🪚 are introduced: add the `breaking change` label
- [ ] If **bugs** 🐞 are fixed: add the `backport <release branch>` label
- [ ] The [documentation website](docs) 📚 has received the love it deserves

### CODE REVIEW

File: libs/feature/editor/src/lib/components/record-form/form-field/form-field.component.html [hunk 1]
```diff
@@ -66,6 +66,12 @@
         (valueChange)="valueChange.emit($event)"
       ></gn-ui-form-field-overviews>
     </ng-container>
+    <ng-container *ngSwitchCase="'resourceCreated'">
+      <gn-ui-form-field-date-updated
```
Comment (Angi-Kinas) [inline libs/feature/editor/src/lib/components/record-form/form-field/form-field.component.html:70]: This does not have to be changed, but I was wondering if the name of this component is too specific as it serves date updated and date created now.

**Review by Angi-Kinas — APPROVED** [review]
Thanks @ronitjadhav for adding this field!

The code looks good and it works as expected :rocket: 

I have some minor comments, you can decide if you want to change something.

File: libs/feature/editor/src/lib/components/record-form/form-field/form-field.component.html (hunk 1 above)
Comment (ronitjadhav) [reply libs/feature/editor/src/lib/components/record-form/form-field/form-field.component.html:70]: I was also thinking of the same.
I guess I can change it to `gn-ui-form-field-date`

**Review by Angi-Kinas — COMMENTED** [review]
Looks good :+1:

**Pull request merged**